StreemNow/LoginViewController.swift

```diff
@@ -37,8 +33,65 @@ class LoginViewController : UIViewControllerSupport {
                 self.present(alert, animated: true)
             }
         }
+
+        // Login to the app's server, and obtain a Streem Token (wrapped in a StreemIdentity)...
+        loginToAppServer(withCompanyCode: companyCode) { [weak self] companySupportsOpenId, streemIdentity, errorMessage in
+            guard let self = self else { return }
+
+            // ...and then identify the user to Streem's server:
+            if let streemIdentity = streemIdentity {
+                print("Successfully logged in: \(streemIdentity)")
+
+                Streem.sharedInstance.identify(with: streemIdentity) { success in
+                    self.showActivityIndicator(false)
+
+                    if success {
+                        self.defaults.set(companyCode, forKey: "companyCode")
+                       
+                        DispatchQueue.main.async {
+                            self.navigationController?.popViewController(animated: true)
+                        }
+                    } else {
+                        showFailure()
+                    }
+                }
+            }
```

Comment:
@jgrund01 looks like you chopped out too many lines here. You still need an `else` clause to handle failed logins:
```
            } else {
                DispatchQueue.main.async {
                    self.showActivityIndicator(false)
                     self.showFailure(errorMessage)
                }
            }
```